cartographer/mapping_3d/sparse_pose_graph.cc

```diff
@@ -190,7 +196,8 @@ void SparsePoseGraph::ComputeConstraint(const mapping::NodeId& node_id,
         &trajectory_nodes_.at(node_id.trajectory_id)
              .at(node_id.node_index)
              .constant_data->range_data_3d.returns,
-        submap_nodes, &trajectory_connectivity_);
+        submap_nodes, initial_relative_pose.rotation(),
```

Comment:
I don't get this... Could you add a comment why `initial_relative_pose` is the gravity orientation.